IntegrationTests/Sources/IntegrationTestSupport/Helpers.swift

```diff
@@ -1,49 +1,51 @@
 /*
-This source file is part of the Swift.org open source project
+ This source file is part of the Swift.org open source project
 
-Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
-Licensed under Apache License v2.0 with Runtime Library Exception
+ Copyright (c) 2014 - 2025 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
 
-See http://swift.org/LICENSE.txt for license information
-See http://swift.org/CONTRIBUTORS.txt for Swift project authors
-*/
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
 
 import Foundation
-import XCTest
 import TSCBasic
 import TSCTestSupport
+import XCTest
```

Comment:
issue (blocking): Since the `IntegrationTests` have been converted to Swift Testing, can we remove this import once we convert the `XCTFails` to the appropriate Swift Testing equivalent, which is posted in another comment.